packages/types/src/documents/app/automation.ts

```diff
@@ -122,6 +123,7 @@ export interface AutomationStepSchema {
     outputs: InputOutputBlock
   }
   custom?: boolean
+  canLoop?: boolean
```

Comment:
As with other comment, this could be adjusted to `features: AutomationFeature[]` and then create the automation feature enum.